osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -144,10 +144,6 @@ private double strainValueOf(DifficultyHitObject current)
             if (osuPrevObj != null && strainTime < greatWindowFull && osuPrevObj.StrainTime > strainTime)
                 strainTime = Interpolation.Lerp(osuPrevObj.StrainTime, strainTime, speedWindowRatio);
 
-            // Cap deltatime to the OD 300 hitwindow.
-            // 0.93 is derived from making sure 260bpm OD8 streams aren't nerfed harshly, whilst 0.92 limits the effect of the cap.
-            strainTime /= Math.Clamp((strainTime / greatWindowFull) / 0.93, 0.92, 1);
```

Comment:
Ditto

Reply:
This was intentional, but could be reverted.